device/celestica/x86_64-cel_silverstone-r0/sonic_platform/helper.py

```diff
@@ -13,7 +14,7 @@ def __init__(self):
         pass
 
     def is_host(self):
-        return os.system(HOST_CHK_CMD) == 0
+        return subprocess.call(HOST_CHK_CMD) == 0
```

Comment:
Please consider changing the is_host function implementation as suggested above.